src/config/config.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
```

Comment:
to not use @Injectable(), here you can create new instance of ConfigService and use it everywhere, because all methods are static